browser/templates/browser/person.html

```diff
@@ -143,8 +156,18 @@
                     {% for research in researches %}
                     <tr>
                         <td>
-                            <a class="btn btn-xs fa fa-edit"
-                                aria-hidden="true">
+                            <a class=" btn btn-xs fa fa-edit"
+                               aria-hidden="true"
+                               href="{% url "edit-investigator" person.id research.id %}">
+                            </a>
+                            <a class="open-InvestigatorDeleteModal btn btn-xs fa fa-trash"
```

Comment:
why is ```open-InvestigatorDeleteModal``` a class and not an id?

Reply:
Changed open-InvestigatorDeleteModal to id